contracts/land/LANDRegistry.sol

```diff
@@ -319,6 +319,7 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
 
     for (uint i = 0; i < x.length; i++) {
       uint256 tokenId = _encodeTokenId(x[i], y[i]);
+      require(_isAuthorized(msg.sender, tokenId));
       _moveToken(
```

Comment:
I think it might be best to send `msg.sender` to the `_moveToken(` function as it already has a modifier `isCurrentOwner(from, assetId)`. What do you think?

Reply:
The main purpose of checking if it is authorized here if related of what I said offline. 
*An operator can create an Estate*

Reply:
Yes, thanks @nachomazzara !

Reply:
`_doTransferFrom` checks if the user is authorized using the modifier `onlyAuthorized`, then calls `_moveToken` that checks `isCurrentOwner(from, assetId)` which will be fail as you are no the owner (if operator)